2023/24xxx/CVE-2023-24012.json

```diff
@@ -1,18 +1,102 @@
 {
```

Comment:
CNA(s) of user (Alias) do not match owning CNA.

Reply:
See https://github.com/CVEProject/cvelist/pull/8694#issuecomment-1445044461